cscs-checks/tools/profiling_and_debugging/gdb4hpc.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class SbucheckCommandCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.valid_systems = ['daint:login', 'dom:login']
+        self.descr = 'Cray parallel debugger'
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.modules = ['gdb4hpc']
+
+        self.executable  = 'gdb4hpc -v'
+        self.executable += ';cd F90; pwd'
+        self.executable += ";make -j 1 CC='cc' CXX='CC' FC='ftn' NVCC='nvcc' CFLAGS='-g -O2 -fopenmp' CXXFLAGS='-g -O2 -fopenmp' FCFLAGS='-g -O2 -fopenmp'"
+        self.executable += ';ls -lrt'
+        self.executable += ';gdb4hpc -b ./gdb4hpc.in &> gdb4hpc.rpt'
```

Comment:
@jgphpc, I see the problem you are facing here. Can you try to use to use an empty launcher?

Reply:
9abcc8611d289afa0fd0e4da131830471b1a8362